libs/ledger/src/executor.cpp

```diff
@@ -49,36 +49,26 @@ namespace fetch {
 namespace ledger {
 namespace {
 
-bool GenerateContractName(chain::Transaction const &tx, Identifier &identifier)
+bool GenerateContractName(chain::Transaction const &tx, ConstByteArray &identifier)
```

Comment:
This function can no longer return false, so arguably it's no longer pulling its weight - try to get rid of it?